testData/completion/fieldAnnotated.py

```diff
@@ -2,12 +2,15 @@
 from typing import Annotated
 from pydantic import BaseModel, Field
 
-
+class Info:
+    pass
 class A(BaseModel):
     abc: Annotated[str, Field(example='example')]
     cde: Annotated[str, Field(example='example')] = 'default_value'
     efg: Annotated[str, Field(default_factory=lambda: 123)]
     a_id: Annotated[str, Field(alias='alias_a_id')]
+    klm: Annotated[str, Info(), Field(default_factory=lambda: 456)]
+    nop: Annotated[str, Field(default_factory=lambda: 789), Info()]
```

Comment:
@Tshimanga 
I expect only `default_factory`. because, pydantic v1 don't allow `default` in the annotated Field.
But, Pydantic V2 doesn't say any error. I will fix it in another PR.
![image](https://github.com/koxudaxi/pydantic-pycharm-plugin/assets/630670/704b9795-e2b2-428f-885e-6189ff8290f9)

Reply:
Thank you!!